inline-caching/README.md

```diff
@@ -0,0 +1,247 @@
+# Inline Cache with GemFire
+
+Using an inline cache with a database can help improve performance, reduce database scaling limits, and provide greater flexibility in terms of data retrieval. 
+It is an important tool for any application that relies heavily on database access, and can help ensure that critical data is always available when it is needed.
+
+GemFire's inline cache is a mechanism for temporarily storing frequently accessed data in memory, rather than querying a database every time the data is needed. 
+This can significantly improve the performance of database-driven applications, especially when dealing with large amounts of data or complex queries.
+
+One of the main reasons to use GemFire with a database is to address scaling limits of the database itself. 
+As the size and complexity of a database grows, it can become increasingly difficult to maintain fast and efficient access to the data. 
+This is especially true in situations where multiple users or applications are simultaneously accessing the database, leading to contention and slowdowns.
+
+By using GemFire's inline cache, frequently accessed data can be stored in GemFire's memory, reducing the number of queries that need to be made 
+to the database. This can help alleviate some of the scaling limits of the database, as it reduces the amount of load placed on the database server. 
+It can also help reduce network latency and improve overall system performance, as data can be retrieved locally rather than over a network connection.
+
+# Implementation
+Setting up an inline cache with GemFire and a database involves several steps. Here's an overview of the process.
+
+## Install and Configure GemFire
+
+The first step is to install and configure GemFire on your system.
+Download and install VMware GemFire from [Tanzu Network](https://network.pivotal.io). Follow the installation instructions in the [GemFire documentation](https://docs.vmware.com/en/VMware-GemFire/index.html).
+
+## Clone the GemFire examples repository from GitHub.
+
+There is a working code examples for how to setup an inline cache here.
+```text
+$ git clone git@github.com:gemfire/gemfire-examples.git
+```
+
+## Configure Access to GemFire Maven Repository
+
+The quickstart tutorial requires access to the VMware Commercial Maven Repository for the GemFire product jars. Please sign-up for access to repo at <https://commercial-repo.pivotal.io/register>.
+
+Create a `gradle.properties` file in this directory with the following information:
```

Comment:
What is "this" directory? Additionally, maybe we could just include a gradle.properties file in the example already with placeholder text

Reply:
We can do the placeholder if you really want. 
It's dangerous though because this is where secure credentials are. By omitting the file, i can tell git never to add it. 

For now, I edited the text to say "root folder of this project"

Reply:
Sounds good! I think thats good enough. Thank you.